src/components/LayerCollection/LayerPOLine.js

```diff
@@ -186,10 +189,18 @@ function LayerPOLine({
         type: 'success',
       });
 
+      const pathname = isCreateAnotherChecked
+        ? `/orders/view/${id}/po-line/create`
+        : `/orders/view/${id}/po-line/view/${savedLine.id}`;
+      const state = isCreateAnotherChecked ? { isCreateAnotherChecked: true } : {};
+
       history.push({
-        pathname: `/orders/view/${id}/po-line/view/${savedLine.id}`,
+        pathname,
         search,
+        state,
       });
+      setSavingValues();
```

Comment:
is it still required?

Reply:
yep. otherwise the form will not be cleared after saving